internal/dependency/manager.go

```diff
@@ -10,6 +10,8 @@ import (
 	"os"
 	"path/filepath"
 
+	"golang.org/x/crypto/ssh/terminal"
```

Comment:
how about using golang.org/x/term instead? The package you use uses golang.org/x/term anyway but the dependency will be smaller

Reply:
Binary size when i use:
golang.org/x/term - 11273644
golang.org/x/crypto/ssh/terminal - 11273612

Reply:
It does: https://github.com/golang/crypto/blob/master/ssh/terminal/terminal.go#L14

I guess `go mod download` is faster when you use golang.org/x/term directly.